cloud-sql/r2dbc/src/main/java/com/example/cloudsql/r2dbcsample/CommanLineRunnerSchemaCreator.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.cloudsql.r2dbcsample;
+
+import io.r2dbc.spi.ConnectionFactory;
+import java.util.Arrays;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.data.r2dbc.core.DatabaseClient;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CommanLineRunnerSchemaCreator implements CommandLineRunner {
```

Comment:
Does this class need to be mentioned in the README?
You don't talk about creating the schema there.

Reply:
@meltsufin It creates schema automatically when the app starts. Could you clarify why you think it should be mentioned in the readme?

Reply:
I see. I guess you can mention that the schema for the `vote` table will be automatically created the first time you run the sample.